x-pack/test/api_integration/apis/ml/trained_models/get_model_stats.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common_api';
+
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  describe('GET trained_models/_stats', () => {
```

Comment:
What happens if the model doesn't exist? Does this work if user doesn't have permission to a space?

Reply:
If it doesn't exist it returns a 404 response, I will add this test case. 
Trained models are not space aware.

Reply:
Added test for 404 response in https://github.com/elastic/kibana/pull/104819/commits/eb7f505eb2ffe3cac6d7c65b60cc5d6a6c4d39c7